index.js

```diff
@@ -0,0 +1,16 @@
+"use strict"
+
+module.exports = {
```

Comment:
We should make eslint configurations for TS and JS consistent. We already have JS configuration under the keep-network org: https://github.com/keep-network/eslint-config-keep. 

We probably need to move `@keep-network/eslint-config-keep` to become `@thesis-co/eslint-config`.

Right now we have:
- `@keep-network/eslint-config-keep` extends `google`,
- `@thesis-co/eslint-config-typescript` extends `airbnb`.

I don't have strong preferences between `airbnb` or `google`, but we should decide on one, unless we intentionally want to use them the way we do right now.

Maybe we could make it even more elegant by implementing what @mhluongo suggested in https://github.com/thesis/eslint-config-typescript/pull/1#discussion_r717558444 (`@thesis-co/eslint-config-typescript` extends `@thesis-co/eslint-config`).

Reply:
I also don't have a strong preference regarding what linting standards we use. The initial commit by Matt was with `airbnb` and we can leave it. After having this discussion, I think we're going to import this repo only for new projects. "Older" keep-* projects will still use the existing `@keep-network/eslint-config-keep` config.

Regarding extending `@thesis-co/eslint-config` I'll see what can be done. `airbnb-typescript` needs `airbnb` to work. If we can avoid duplication, then yea, we can probably extend it.